### DISCUSSION TITLE
Tips for optimizing font size?

### DISCUSSION THREAD

**Discussion by anaclumos** [vercel/satori#434] [Q&A] [answered]
I added a local font, and now it exceeds the limit. Is there a better way of handling this?

![image](https://user-images.githubusercontent.com/31657298/230518227-a680f729-dcb2-4daf-832c-48439ebb9462.png)

**Comment by shuding** [comment, marked as answer]
Try the woff version with a reasonable charset (e.g. Latin) only?

Another way is to serve the font in /public and fetch it on the fly.

**Reply by anaclumos** [reply]
> serve the font in /public and fetch it on the fly.

This method worked. Thanks.
